### ISSUE TITLE
no friends after following README

### ISSUE THREAD

**Issue by myf** [MaskRay/wechatircd#24]
I am using Arch Linux, followed the instruction (the one installs wechatircd using yaourt), used greasemonkey to inject using firefox, used weechat as irc server connected to 127.1/6667. 

The problem is that I can only see myself on the raster without any friends. The weechat message i got was the following
```
         │19:08:45  wechat   -- | Hi, welcome to IRC
         │19:08:45  wechat   -- | Your host is wechatircd.maskray.me
         │19:08:45  wechat   -- | There are 1 users and 0 WeChat users (local to you) on 1 server
```

how should I debug from here? am I missing something crucial?

**Comment by MaskRay** [comment]
- Press F12 to open the devtools & select the `Network` panel
- Reload https://wx.qq.com

Can you find a request to https://127.0.0.1:9000/injector.js ?

**Comment by myf** [comment]
@MaskRay  there is a get event to the address
```
GET 
https://127.0.0.1:9000/injector.js
```
that returns a 200

**Comment by myf** [comment]
there's also a bunch of GETs to `https://127.0.0.1:9000/ws` that returns 101

**Comment by myf** [comment]
ah here
```
The connection to wss://127.0.0.1:9000/ws was interrupted while the page was loading.
```

**Comment by MaskRay** [comment]
It may be the issue of the version of aiohttp

I have
archlinuxcn/python-aiohttp 1.1.5-1
archlinuxcn/python-multidict-git 2.1.3-1
installed on my laptop/vps and they seem to cooperate well.

**Comment by myf** [comment]
that could be the case;

I haved installed these packages from aur
local/python-aiohttp 1.0.5-1
local/python-multidict 2.1.2-1

maybe i'll try to git clone the project instead

**Comment by MaskRay** [comment]
`journalctl -b -u wechatircd`

**Comment by myf** [comment]
https://gist.github.com/myf/b58432538849411dfb1821361ef8ec2b

**Comment by MaskRay** [comment]
It think it should be aiohttp's fault ....  As I was frustrated by it a few months ago. My friend has also encountered this issue recently.

The easiest workaround is to set up archlinuxcn repository and install python-aiohttp

**Comment by myf** [comment]
as i was just retrying the same script it actually worked this time with no websocket errors at all!

thanks! i might see if this problem is reproducable and try the newer version of python aiohttp

**Comment by MaskRay** [comment]
Thanks. I'd like to know whether your issue is caused by aiohttp

I recalled when multidict (dependency of aiohttp) bumped to 2.0 , aiohttp was broken. There is also some mysterious issue with latest versions of aiohttp .

**Comment by MaskRay** [comment]
https://aur.archlinux.org/packages/python-aiohttp/ has been bumped to 1.1.5

**Comment by myf** [comment]
upgraded, there's some dependency issue with the python-aiohttp 1.1.5, with error message 
```
 ImportError: No module named 'yarl'
```
installing `python-yarl` from aur resolves this problem. I think you may add python-yarl as a dependency, or better yet, wait for the `python-aiohttp` owner to add yarl as a dependency

**Comment by MaskRay** [comment]
`python-yarl` has been added as a dependency to `python-aiohttp` https://aur.archlinux.org/packages/python-aiohttp/

**Comment by myf** [comment]
A couple of empirical observations (not knowing the reason) when I tried to set up wechatircd on my server, the friends list do not populate unless i do the following things:

1. in the userscript, add `https://web.wechat.com/*` as domain in addition to `https://wx.qq.com` since the latter really just redirects to the former. This makes sense to me.

2. even when I did that, the connection did not establish. However, when I turned on the inspector `Ctrl-Alt-j` and rerun the sign-in process it all works out flawlessly. I have no explaination about this...
